### PR TITLE
errors store added

### PR DESCRIPTION
**Pull request by romsar**

### CODE REVIEW

File: response.go [hunk 1]
```diff
@@ -218,13 +218,30 @@ func (i *Inertia) htmlContainer(pageJSON []byte) (inertia, _ template.HTML, _ er
...
 	if IsInertiaRequest(r) {
+		i.captureValidationErrors(r)
 		setInertiaLocationInResponse(w, url)
 		return
 	}
 
 	redirectResponse(w, r, url, status...)
 }
 
+func (i *Inertia) captureValidationErrors(r *http.Request) {
```
Comment (JensvandeWiel) [inline response.go:229]: Right now it returns early if there is no errorStore. Maybe it's smart to still check and add the errors from context before checking if there is an errorStore

**Comment by JensvandeWiel** [comment]
If i may ask, why was this closed?

File: response.go (hunk 1 above)
Comment (romsar) [reply response.go:229]: Sorry, I rethought some ideas and it seemed easier to start over.

Comment (JensvandeWiel) [reply response.go:229]: @romsar I just saw, I think that is the correct way, not forcing people to use it but add the possibility.

**Pull request closed without merging**